[2025-03-01T07:00:04.800+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-03-01T06:57:00+00:00 [queued]>
[2025-03-01T07:00:04.810+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-03-01T06:57:00+00:00 [queued]>
[2025-03-01T07:00:04.811+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-03-01T07:00:04.825+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): load_to_postgres_file> on 2025-03-01 06:57:00+00:00
[2025-03-01T07:00:04.831+0000] {standard_task_runner.py:60} INFO - Started process 3510 to run task
[2025-03-01T07:00:04.834+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'crypto_etl_dag', 'load_to_postgres_file', 'scheduled__2025-03-01T06:57:00+00:00', '--job-id', '353', '--raw', '--subdir', 'DAGS_FOLDER/crypto_etl_dag.py', '--cfg-path', '/tmp/tmputuczr8s']
[2025-03-01T07:00:04.838+0000] {standard_task_runner.py:88} INFO - Job 353: Subtask load_to_postgres_file
[2025-03-01T07:00:04.896+0000] {task_command.py:423} INFO - Running <TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-03-01T06:57:00+00:00 [running]> on host 335ace54b661
[2025-03-01T07:00:04.976+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_etl_dag' AIRFLOW_CTX_TASK_ID='load_to_postgres_file' AIRFLOW_CTX_EXECUTION_DATE='2025-03-01T06:57:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-01T06:57:00+00:00'
[2025-03-01T07:00:04.979+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-01T07:00:04.980+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/scripts/load_to_postgres.py']
[2025-03-01T07:00:04.993+0000] {subprocess.py:86} INFO - Output:
[2025-03-01T07:00:06.003+0000] {subprocess.py:93} INFO - [INFO] Đã tải file '2025/03/01/14-00-02.csv' từ MinIO
[2025-03-01T07:00:06.004+0000] {subprocess.py:93} INFO - [INFO] Đã nạp 10 bản ghi vào Data Warehouse thành công.
[2025-03-01T07:00:06.005+0000] {subprocess.py:93} INFO - [INFO] Quá trình ETL hoàn tất. Đã xử lý 1 file mới từ MinIO.
[2025-03-01T07:00:06.075+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-01T07:00:06.097+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=crypto_etl_dag, task_id=load_to_postgres_file, execution_date=20250301T065700, start_date=20250301T070004, end_date=20250301T070006
[2025-03-01T07:00:06.133+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-01T07:00:06.153+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
